### PR TITLE
Upgrade to doxia 1.8

### PR DESCRIPTION
**Pull request by Tibor17**
PR for the issue #217 .

### CODE REVIEW

**Comment by Tibor17** [comment]
I applied Doxia 1.8 and fixed the import but I do not know what it has to do with newly failed IT `ISSUE-80` where `<file>` is missing in `licenses.xml`, see `log.txt` :
```
[WARNING] Unable to retrieve license for dependency: org.jboss.xnio:xnio-api
[WARNING] http://repository.jboss.org/licenses/cc0-1.0.txt
[WARNING] connect timed out
```
Second IT which has failed with Doxia 1.8 is `download-licenses-include-exclude-types` maybe with the same root cause:
`license-maven-plugin\target\it\download-licenses-include-exclude-types\target\generated-resources\licenses2\lgpl - lgpl-2.1.txt file is missing or a directory.`

**Comment by Tibor17** [comment]
@ppalaga 
Doxia is dependent plexus. Perhaps more plexus artifacts have to be excluded.

File: pom.xml [hunk 1]
```diff
@@ -116,7 +116,8 @@
 
     <processorVersion>1.3</processorVersion>
 
-    <doxiaVersion>1.6</doxiaVersion>
+    <doxiaVersion>1.8</doxiaVersion>
+    <doxiaRendererVersion>1.8.1</doxiaRendererVersion>
```
Comment (ppalaga) [inline pom.xml]: If we really need both `doxiaVersion` and `doxiaRendererVersion` please add a comment explaining why.

Comment (Tibor17) [reply pom.xml]: I think they fixed important stability issues, especially the last two:
https://issues.apache.org/jira/browse/DOXIASITETOOLS-187
https://issues.apache.org/jira/browse/DOXIASITETOOLS-179
https://issues.apache.org/jira/browse/DOXIASITETOOLS-189

Comment (ppalaga) [reply pom.xml]: So `org.apache.maven.doxia:doxia-site-renderer` has independent release cycle from other `org.apache.maven.doxia:*` artifacts that we use? If so, this is what should be noted on line 120 of our pom.xml.

File: pom.xml [hunk 2]
```diff
@@ -116,7 +116,9 @@
 
     <processorVersion>1.3</processorVersion>
 
-    <doxiaVersion>1.6</doxiaVersion>
+    <doxiaVersion>1.8</doxiaVersion>
+    <!-- ASF project maven-doxia-sitetools:1.8.1 has different release cycle and it is dependent on maven-doxia:1.8 -->
```
Comment (Tibor17) [inline pom.xml]: @ppalaga here is the comment

Comment (Tibor17) [reply pom.xml]: We can see the Doxia projects on GitHub, namely doxia and renderer:
https://github.com/apache/maven-doxia
https://github.com/apache/maven-doxia-sitetools

**Comment by ppalaga** [comment]
Thanks @Tibor17 !

**Comment by Tibor17** [comment]
@ppalaga Thx, would you cut a release version on this?

**Comment by ppalaga** [comment]
> would you cut a release version on this?

1.17 is being voted on since yesterday. No plans for the next release yet.

**Pull request merged**